### ISSUE TITLE
ChoiceQuestion breaks console output

### ISSUE THREAD

**Issue by ethanclevenger91** [symfony/symfony#37155]
**Symfony version(s) affected**: 5.1.0 (installed via Laravel 7.14.1)

Git/Git Bash 2.27.0
Win 10

**Description**  
Asking a choice question subsequently breaks console formatting in Git Bash on Windows.

**How to reproduce**  
[Started via a Laravel issue](https://github.com/laravel/framework/issues/33119) and was referred here. Don't use a lot of vanilla Symfony, but I got the following to reproduce from a Symfony console command:

```php
$helper = $this->getHelper('question');
$question = new ChoiceQuestion('Select', ['one', 'two'], 0);

$selection= $helper->ask($input, $output, $question);

$output->writeln(['<info>Hey there.</>',]);
```

Outputs: 
![83907383-2b8ef980-a71a-11ea-8c1d-a171a16a17d6](https://user-images.githubusercontent.com/2623646/84056309-e57ca480-a96a-11ea-8591-e2b093fe4ceb.png)

**Comment by carsonbot** [comment]
Hey, thanks for your report!
There has not been a lot of activity here for a while. Is this bug still relevant? Have you managed to find a workaround?

**Comment by ethanclevenger91** [comment]
Have not found a workaround. Also have not tested latest version of Symfony.

**Comment by NicRoss-COM** [comment]
This is still relevant and reproducible in Symfony 5.3.3 with git bash. There is currently no workaround beyond avoiding [`Symfony\Component\Console\Question\ChoiceQuestion`](https://symfony.com/doc/current/components/console/helpers/questionhelper.html#let-the-user-choose-from-a-list-of-answers) and [color styles](https://symfony.com/doc/current/console/coloring.html). It's especially illegible when the console outputs an exception.

**Comment by GaryCXJk** [comment]
After doing some debugging myself, I found that `Terminal::hasSttyAvailable()` is the reason it all breaks down, specifically, line 72:

```php
        exec('stty 2>&1', $output, $exitcode);
```

Apparently even just running exec will break colors. I don't think there's a good solution for this, other than to just not execute commands.

EDIT:

This seems to be only a problem when using xterm. I changed my terminal to vt100, which solved the problem. This is not a Symfony issue, at the very least.
